src/features/ajax/aggregate/gql.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @typedef {object} GQLMetadata
+ * @property {string} operationName Name of the operation
+ * @property {string} operationType Type of the operation
+ * @property {string} operationFramework Framework responsible for the operation
+ */
+
+/**
+ * @param {string|Object} gql The GraphQL object body sent to a GQL server
+ * @returns {GQLMetadata}
+ */
+export function parseGQL (gql) {
+  if (!gql) return
+  const contents = gql
```

Comment:
Is this an unnecessary variable creation?

Reply:
yes, will clean up ty